### ISSUE TITLE
Error when installing and loading DuckDB extensions like postgres_scanner

### ISSUE THREAD

**Issue by BigBerny** [TablePlus/TablePlus#3363]
### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
DuckDB 0.10.3 (1.0.0 would be nice)

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
568

3. The steps to reproduce this issue:
Open DuckDB table and run:
INSTALL postgres_scanner;
LOAD postgres_scanner;

**Comment by huyphams** [comment]
Hi @BigBerny for security reason, TablePlus could not load the dynamic libs which are not signed by TablePlus team, so it won't work.

**Comment by BigBerny** [comment]
Anything that can be done here? Many of the extensions are core extensions from DuckDB and maintained by DuckDB: https://duckdb.org/docs/extensions/core_extensions.html
Without these extensions many features won't work. For example you can't access Parquet files on S3 which is a very common use case.

**Comment by tim-x-y-z** [comment]
second @BigBerny would be really good to have at least the core extensions available in table plus.

**Comment by choval** [comment]
> The issue in TablePlus is not only related to the extension `postgres_scanner`but also to `httpfs` and other extensions which are not distributed built-in (details see https://duckdb.org/docs/extensions/core_extensions.html#default-extensions).
> 
> Currently there seem to be only the following built-in extensions available in TablePlus (`SELECT * FROM duckdb_extensions();`):
> 
> * icu
> * json
> * parquet
> 
> I tested the same INSTALL and LOAD commands in DBeaver without any problems. I don't know why, but DBeaver can load the dynamic libs. Do they use the same signing keys?
> 
> Would it be possible to have more built-in extensions available in TablePlus? Yes, these extensions cannot be updated with the INSTALL / LOAD commands. But using not-so-current extensions is better than no extension at all.

Agree! At least give us all the built-in/core extensions. Would a custom extensions path, with the core extensions installed work to avoid the shared extension path that have the tableplus unsigned ones?

**Comment by jovezhong** [comment]
There are some cool features in DuckDB recently such as Iceberg support. It'll be awesome to load those extensions, at least the core ones.

**Comment by theelderbeever** [comment]
Any movement on this? Not being able to read from object storage via `httpfs` is a major hit on functionality of duckdb. The lack of extensions makes duckdb a nearly pointless connection for TablePlus to support.

**Comment by ernestyao** [comment]
Excel extension is not supported by TablePlus 670. I suppose it's about the version of duckdb engine within TablePlus. It's 1.6.0dev, a development version in stead of a stable release. So some extensions cannot be found in official site. I cannot understand this decision. Please add a setting to give us the choice.

**Comment by huyphams** [comment]
Hi guys, I’m sorry for the late reply, but the issue has nothing to do with DuckDB. It is a TablePlus issue where the app is too strict about loading dynamic libraries that are not part of the app bundle (which is notarized). We did this in the first place to ensure the highest level of security.

We are thinking about removing this limitation in the upcoming releases.

I hope you understand the reason behind this. It is not a bug, but a decision.

**Comment by huyphams** [comment]
Hi, we enabled the dylib side-loading, you can try the build here, 
https://www.dropbox.com/scl/fi/mpnhecs75ruoqe3442fpa/TablePlus.dmg?rlkey=594kh1wtsg85zmqcslim6jitj&dl=0

**Comment by huyphams** [comment]
Hi @ernestyao DuckDB-Swift does not have the official release for the stable 1.5.2 let me see if we can fork it and sync it

**Comment by ernestyao** [comment]
> Hi [@ernestyao](https://github.com/ernestyao) DuckDB-Swift does not have the official release for the stable 1.5.2 let me see if we can fork it and sync it ref: https://github.com/duckdb/duckdb-swift

I installed duckdb from Homebrew: `brew install duckdb`. And then `duckdb --version` says it's "v1.5.2 (Variegata) 8a5851971f". It works well with TablePro and DBeaver.

**Comment by huyphams** [comment]
Hi @ernestyao if you have a chance, please try this patch 
https://www.dropbox.com/scl/fi/mpnhecs75ruoqe3442fpa/TablePlus.dmg?rlkey=594kh1wtsg85zmqcslim6jitj&dl=0

We've synced the version 1.5.2 (stable release) and allow side-loading the dynamic lib.

So I'm closing this issue.

**Comment by kinghuang** [comment]
I just tried this build and was able to load and install the `postgres_scanner` and `ducklake` extensions.
